lib/browser/guest-window-manager.js

```diff
@@ -219,7 +219,7 @@ const canAccessWindow = function (sender, target) {
 };
 
 // Routed window.open messages with raw options
-ipcMainInternal.on('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_OPEN', (event, url, frameName, features) => {
+ipcMainInternal.on('ELECTRON_GUEST_WINDOW_MANAGER_WINDOW_OPEN', async (event, url, frameName, features) => {
```

Comment:
This has to be a sync IPC message for _other reasons_ and it doesn't line up with the spec to block a `window.open` call until the child window has opened.  What exactly are we trying to solve here?

Reply:
You essentially can't use any of the properties in `window.location` the way it's written. They will all throw errors.

Reply:
@codebytere So let's do a `''` check on that `getURL()` result and report the location as `about:blank` instead.  That's what the browser does

![image](https://user-images.githubusercontent.com/6634592/82938300-705ea780-9f46-11ea-8abc-6e19ab179dc2.png)